pom.xml

```diff
@@ -600,6 +608,7 @@
             <id>mysql</id>
             <properties>
                 <database.container>mysql</database.container>
+                <osiam.test.host>${osiam.host.protocol}://${osiam.host}</osiam.test.host>
```

Comment:
I don't see why we need this property in 2 different profiles that always have the same value regardless of the active profile.